modules/machineset-yaml-azure.adoc

```diff
@@ -26,71 +26,71 @@ apiVersion: machine.openshift.io/v1beta1
+    machine.openshift.io/cluster-api-machine-type: infra
+  name: <infrastructure_id>-infra-<region> # <3>
 endif::infra[]
   namespace: openshift-machine-api
 spec:
   replicas: 1
   selector:
     matchLabels:
-      machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
 ifndef::infra[]
-      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<region> <3>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<region>
 endif::infra[]
 ifdef::infra[]
-      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-infra-<region> <3>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-infra-<region>
 endif::infra[]
   template:
     metadata:
       creationTimestamp: null
       labels:
-        machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
 ifndef::infra[]
-        machine.openshift.io/cluster-api-machine-role: <role> <2>
-        machine.openshift.io/cluster-api-machine-type: <role> <2>
-        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<region> <3>
+        machine.openshift.io/cluster-api-machine-role: <role>
+        machine.openshift.io/cluster-api-machine-type: <role>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<region>
 endif::infra[]
 ifdef::infra[]
-        machine.openshift.io/cluster-api-machine-role: <infra> <2>
-        machine.openshift.io/cluster-api-machine-type: <infra> <2>
-        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-infra-<region> <3>
+        machine.openshift.io/cluster-api-machine-role: infra
+        machine.openshift.io/cluster-api-machine-type: infra
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-infra-<region>
 endif::infra[]
     spec:
       metadata:
         creationTimestamp: null
         labels:
           machine.openshift.io/cluster-api-machineset: <machineset_name>
 ifndef::infra[]
-          node-role.kubernetes.io/<role>: "" <2>
+          node-role.kubernetes.io/<role>: ""
 endif::infra[]
 ifdef::infra[]
-          node-role.kubernetes.io/infra: "" <2>
+          node-role.kubernetes.io/infra: ""
 endif::infra[]
       providerSpec:
         value:
           apiVersion: azureproviderconfig.openshift.io/v1beta1
           credentialsSecret:
             name: azure-cloud-credentials
             namespace: openshift-machine-api
-          image: <4>
+          image: # <4>
             offer: ""
             publisher: ""
-            resourceID: /resourceGroups/<infrastructure_id>-rg/providers/Microsoft.Compute/images/<infrastructure_id> <5>
+            resourceID: /resourceGroups/<cluster_id>-rg/providers/Microsoft.Compute/galleries/gallery_<cluster_id>/images/<cluster_id>-gen2/versions/latest # <5>
```

Comment:
seems we have been using <infrastructure_id> instead of <cluster_id>

Reply:
Thanks! I'll get that one here, and fix other instances in docs in a separate PR.